core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -1422,6 +1500,7 @@ export class WorkflowEditorComponent implements AfterViewInit, OnDestroy {
    * @param copiedOperatorID
    * @param positions
    */
+  // private calcOperatorPosition(newOperatorID: string, copiedOperatorID: string, positions: Point[]): Point {
```

Comment:
Please only keep the new method. You can directly modify the old method if that's not useful anymore.